python/README.md

```diff
@@ -1,14 +1,11 @@
 # CSXCAD python interface
 
 ## Install
-* Simple version:
-```python
-python setup.py install
-```
+If openEMS was installed into `~/opt/openEMS`, then install this package with:
 
-* Extended options, e.g. for custom install path at */opt/openEMS*:
 ```python
-python setup.py build_ext -I/opt/openEMS/include -L/opt/openEMS/lib -R/opt/openEMS/lib"
+python setup.py build_ext -I ~/opt/openEMS/include -L ~/opt/openEMS/lib -R ~/opt/openEMS/lib"
 pyhton setup.py install
 ```
-**Note:** The install command may require root on Linux, or add --user to install to ~/.local
```

Comment:
This note is still valid? Ideally this should be done in a virtualenv?

Reply:
It is a python package much like any other. Under normal conditions:
- No need to require root privileges,
- `--user` is completely optional, and
- installing it inside a virtual environment is also optional, like any other package. However, it is recommended (as for any other package). Even more, some newer distributions like Ubuntu 24.04 push you in that direction, and by default do not let you install packages outside a virtualenv anymore. But again, this is the same for any python package, be it this one, numpy or matplotlib.

Reply:
So you mean we should remove this note as it is true for any package?
This is true, but still this might help to remind people what to do if install fails?

Reply:
I would do what most standard Python packages do. I.e., remove the note.

The best is go for `pip` installation. This package fails if you try to go with `pip`, if I have time I will try to fix that so it becomes trivial to install.